packages/discovery-provider/ddl/migrations/0132_trending_cleanup.sql

```diff
@@ -0,0 +1,9 @@
+begin;
+
+delete from track_trending_scores where version != 'pnagD';
+
+create index ix_trending_scores
+on track_trending_scores (type, version, time_range, score desc)
+include (track_id);
```

Comment:
Interesting, why not have it as part of the index, since it's being used to sort? I guess only marginally impactful vs extra index size?

Reply:
yeah the impact is so low, i dont think it's worth adding to the size. but im happy to add it as well. the include (track_id) part is nice though because it means we can get the results directly from the query instead of hitting the actual db

Reply:
ah i guess it works for index-only if it's part of the key instead of the include. cool will just do that. probs the same

Reply:
yeah i think the difference is kinda moot but was curious